### ISSUE TITLE
Should README default to Lua?

### ISSUE THREAD

**Issue by adrian5** [nvim-treesitter/nvim-treesitter#2503]
At this stage, should we move from `init.vim` with `lua <<EOF` to `init.lua` and give the config examples the `lua` highlight?

**Comment by kyazdani42** [comment]
I agree it'd be cleaner. @vigoux @theHamsta @clason some input on this ? I could just open the PR.

**Comment by clason** [comment]
Just go for it!

**Comment by theHamsta** [comment]
I'm pretty neutral to this, I just want to remember that we did this refactor ago but in the other direction. But I think that with 0.5 published people are more familiar with Lua nowadays.

**Comment by clason** [comment]
I think we edited for consistency back then, and decided to stay "traditional". I'd say things have changed a lot in the meantime to make a Lua-first README more natural.
